OAuth Integrations - Troubleshooting (#25998)
DataDog/documentation: content/en/developers/integrations/oauth_for_integrations.md

For issues with the PKCE OAuth flow, ensure the `content-type` header is correctly set to `application/json` or `application/x-www-form-urlencoded`.

### Regenerating client secrets and secret rotation

> If your secret was leaked and needs to be rotated, contact [ecosystems@datadog.com][11]. Only one secret can be active at a time. Once you regenerate your secret, the existing secret will be deleted. Customers do not need to re-authorize the integration.

Reviewer: ```suggestion
If your secret was leaked and needs to be rotated, contact [ecosystems@datadog.com][11]. Only one secret can be active at a time. After you regenerate your secret, the existing secret is deleted. You do not need to re-authorize the integration.
```
